pkg/machines/components/vmnetworktab.jsx

```diff
@@ -37,11 +38,32 @@ class VmNetworkTab extends React.Component {
         super(props);
 
         this.state = {
+            showModal: false,
             interfaceAddress: [],
+            networkDevices: undefined,
         };
+
+        this.open = this.open.bind(this);
+        this.close = this.close.bind(this);
+    }
+
+    close() {
+        this.setState({ showModal: false });
+    }
+
+    open() {
+        this.setState({ showModal: true });
     }
 
     componentDidMount() {
+        cockpit.spawn(["ls", "/sys/class/net"])
+                .fail(e => console.log(e))
+                .done(output => {
+                    const devs = output.split('\n');
+                    devs.pop();
```

Comment:
Can you leave me a comment why we need the pop here :) I don't really see through it.